api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/selection/SelectedLocale.java

```diff
@@ -24,7 +24,7 @@
 	private static final Log log = LogFactory.getLog(SelectedLocale.class);
 
 	/** Use this attribute on both the ServletContext and the Session. */
-	protected static final String ATTRIBUTE_NAME = "SELECTED_LOCALE";
+	public static final String ATTRIBUTE_NAME = "SELECTED_LOCALE";
```

Comment:
Why has this been made public? I can't see a corresponding use of SELECTED_LOCALE.